### DISCUSSION TITLE
Zee Cinema Middle East

### DISCUSSION THREAD

**Discussion by Rgubds** [iptv-org/community#443] [Show and tell]
### Channel ID

ZeeCinemaMiddleEast.in

### Stream URL

https://y5w8j4a9.ssi.hwcdn.net/zeecinemame/tracks-v1/index.m3u8

### Notes

This channel showcases hindi movies with english subtitles

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Seems it's only available to India.

**Comment by R4IN2007** [comment]
Look's like the m3u8 link doesn't work for me especially the United States

**Comment by rsk1987** [comment]
not working in india

**Comment by BellezaEmporium** [comment]
Another HWCDN host located in https://v4h7x7n8.ssl.hwcdn.net/zeecinemame/playlist.m3u8 gives me 400

**Comment by freearhey** [comment]
https://y5w8j4a9.ssi.hwcdn.net/zeecinemame/tracks-v1/index.m3u8 (offline)
<img width="476" alt="image" src="https://user-images.githubusercontent.com/7253922/232255129-7d2d636f-cfa2-4afa-8d05-eb341f6e3c67.png">

https://v4h7x7n8.ssl.hwcdn.net/zeecinemame/playlist.m3u8 (offline)
<img width="421" alt="image" src="https://user-images.githubusercontent.com/7253922/232255160-b5682324-66b6-49e6-bcfb-9341db90e49d.png">
